firefox-ios/Client/Frontend/Autofill/Address/Edit/EditAddressViewController.swift

```diff
@@ -48,29 +89,34 @@ class EditAddressViewController: UIViewController, WKNavigationDelegate, WKScrip
         do {
             let javascript = try model.getInjectJSONDataInit()
             self.evaluateJavaScript(javascript)
+            if case .add = model.destination {
+                self.evaluateJavaScript("toggleEditMode(true);")
+            }
         } catch {
-            self.logger.log("Error injecting JavaScript",
-                            level: .warning,
-                            category: .autofill,
-                            description: "Error injecting JavaScript: \(error.localizedDescription)")
+            self.logger.log(
+                "Error injecting JavaScript",
+                level: .warning,
+                category: .autofill,
+                description: "Error injecting JavaScript: \(error.localizedDescription)"
```

Comment:
nit: Error evaluating instead of injecting

Reply:
We can actually remove this code.